tests/acceptance/test_v2_measurement_service.py

```diff
@@ -93,62 +72,44 @@ def test___streaming_measurement_service___specify_data_size___receives_expected
     index = 0
     for response in response_iterator:
         expected_data = [index for i in range(data_size)]
-        expected = _get_serialized_measurement_outputs("test", index, expected_data)
+        expected = _get_serialized_measurement_outputs(name, index, expected_data)
         assert expected == response.outputs.value
         index += 1
 
-    assert index == num_responses
-
 
 @pytest.mark.parametrize("error_on_index", [1, 5, 9])
 def test___streaming_measurement_service___specify_error_index___errors_at_expected_response(
     error_on_index: int, stub_v2: v2_measurement_service_pb2_grpc.MeasurementServiceStub
 ):
     """End to End Test validating streaming measurement sends responses prior to expected error."""
-    name = "test"
-    num_responses = 10
-    data_size = 1
-    cumulative_data = True
-    response_interval_in_ms = 1
     request = v2_measurement_service_pb2.MeasureRequest(
-        configuration_parameters=_get_configuration_parameters(
-            name, num_responses, data_size, cumulative_data, response_interval_in_ms, error_on_index
-        )
+        configuration_parameters=_get_configuration_parameters(error_on_index=error_on_index)
     )
 
     response_iterator = stub_v2.Measure(request)
 
     index = 0
```

Comment:
FYI: Python functions have a single scope for local variables. `with` and `for` do not begin a new scope. https://realpython.com/python-scope-legb-rule/

```
with pytest.raises(grpc.RpcError, match=f"Errored at index {error_on_index}"):
    for index, response in enumerate(response_iterator):
        pass

assert index == error_on_index
```